Source/Scene/Points3DTileContent.js

```diff
@@ -403,10 +562,21 @@ define([
             vs += '    vec3 position = a_position; \n';
         }
 
+        // TODO : only for testing
```

Comment:
Ah, look at the shader generation in the Batch Table, but I think we want to prefix these more specifically than `style_`.